repos/spack_repo/builtin/packages/kokkos/package.py

```diff
@@ -398,6 +397,8 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
         sha256="145619e87dbf26b66ea23e76906576e2a854a3b09f2a2dd70363e61419fa6a6e",
         when="@4.2.00",
     )
+    # Remove unnecessary C and C++ languages dependency in scripts/spack_test/CMakeLists.txt
```

Comment:
See above patch for example on how to download and document a patch that was upstreamed. Even if the folder is not part of release tarballs, there is git versions like `develop` that still contain the folder.

Reply:
> See above patch for example on how to download and document a patch that was upstreamed.

see 7686857c5bc27af95cf64f3ef783f3275667171c